addons/mail/static/tests/new/discuss/channel_member_list_tests.js

```diff
@@ -152,4 +152,58 @@ QUnit.module("mail", (hooks) => {
             assert.containsOnce(target, ".o-mail-autoresize-input[title='Demo']");
         }
     );
+
+    QUnit.test(
+        "should show a button to load more members if they are not all loaded",
+        async function (assert) {
+            // Test assumes at most 100 members are loaded at once.
```

Comment:
Just added this comment in tests, so it's clearer why tests are correct with current values.